src/components/CardLoader.tsx

```diff
@@ -13,7 +13,7 @@ const getCardStyle = (pos: { x: number, y: number }) => {
     zIndex: floatNumber,
     opacity: 0,
     top: 0,
-    left: 0
+    left: 0,
```

Comment:
Omit the comma of the last item of an object.

Reply:
Done.